blueprints/ember-redux-shim/index.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
       .then(() => {
         return this.addAddonsToProject({
           packages: [
-            {name: 'ember-lodash-shim', target: '^1.2.0'},
+            {name: 'ember-lodash-shim', target: '^2.0.0'},
```

Comment:
would this require a larger bump as we are pulling in v2.0.0 of the lodash shim? or no because it won't leak outward?